### PR TITLE
tt-torch model test parser

### PR DESCRIPTION
**Pull request by nsmithtt**

### CODE REVIEW

**Comment by nsmithtt** [comment]
@vmilosevic / @AleksKnezevic / @ddilbazTT / @uazizTT

**Review by AleksKnezevic — APPROVED** [review]

File: .github/actions/collect_data/src/parsers/tt_torch_model_tests_parser.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import tarfile
+import os
+import json
+from loguru import logger
+from functools import partial
```
Comment (vmilosevic) [inline .github/actions/collect_data/src/parsers/tt_torch_model_tests_parser.py:9]: Please remove unused imports partial and junit_xml_utils

**Review by vmilosevic — APPROVED** [review]
Can you include test for TTTorchModelTestsParser? Simple test is enough, process one file and assert that number of discovered tests is correct.

**Review by vmilosevic — CHANGES_REQUESTED** [review]
.

**Comment by warthog9** [comment]
No issues from my side

**Review by vmilosevic — APPROVED** [review]

**Comment by vmilosevic** [comment]
@nsmithtt  The error you see in the workflow is because you are making a PR from the fork, and it doesn't have permission to add comments to PR to show the report.

We can ignore this, easiest way to get around this is by adding "continue-on-error: true" to "Show reports" step

File .github/workflows/test_collect_data_action.yml Line 37 add 
continue-on-error: true

**Pull request merged**